lib/phantom-script.js

```diff
@@ -69,8 +69,8 @@ page.open(skeleton, function (status) {
   page.paperSize = defaultFormat;
 
   if (args.runningsPath !== "nofile") {
-    var runningsArgs = JSON.parse(args.runningsArgs);
-    var runnings = require(args.runningsPath)(runningsArgs);
+    var runningsArgs = args.runningsArgs ? JSON.parse(args.runningsArgs) : undefined;
```

Comment:
Empty string is a falsey value though, so when `args.runningsArgs === ''`, this will evaluate to `false` and set `runningsArgs = undefined`